cmd/soroban-rpc/internal/test/cli_test.go

```diff
@@ -20,6 +21,71 @@ import (
 	"gotest.tools/v3/icmd"
 )
 
+func cargoTest(t *testing.T, name string) {
+	NewCLITest(t)
+	c := icmd.Command("cargo", "test", "--package", "soroban-test", "--test", "it", "--", name, "--exact", "--nocapture")
+	c.Env = append(os.Environ(),
+		fmt.Sprintf("SOROBAN_RPC_URL=http://localhost:%d/", sorobanRPCPort),
+		fmt.Sprintf("SOROBAN_NETWORK_PASSPHRASE=%s", StandaloneNetworkPassphrase),
+	)
+	res := icmd.RunCmd(c)
+	require.NoError(t, res.Error, res.Stdout(), res.Stderr())
+}
+
+func TestCLIfetch(t *testing.T) {
+	cargoTest(t, "contract_sandbox::fetch")
+}
```

Comment:
Instead of having a separate test for each rust test please use something like `cargo -q test contract_sandbox:: --package soroban-test -- --list` to get all the test names in conjunction with https://pkg.go.dev/testing#T.Run

Reply:
BTW, `contract_sandbox` should be renamed or the tests moved to a separate namespace since they are not just testing the sandbox anymore,